docs/getting-started-guides/ubuntu.md

```diff
@@ -55,7 +55,7 @@ work, which has been merge into this document.
 
 1. The nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge.
 2. All machines can communicate with each other, no need to connect Internet (should use
-private docker registry in this case).
+private docker registry in this case). **Note:** When deploying, master needs to connect Internet to download the necessary files.
```

Comment:
2. All machines can communicate with each other. Master node needs to connect the Internet to download the necessary files, while working nodes do not.